api/validators.py

```diff
@@ -105,10 +103,10 @@ def g(payload, method):
             try:
                 _validate_json(payload, _schema, resolver_input)
             except jsonschema.ValidationError as e:
-                handler.abort(400, str(e))
+                raise webapp2.exc.HTTPBadRequest(str(e))
```

Comment:
Just pushed a new commit. Added two exceptions to distinguish when the exception is critical(mongo schema validations).